main/webapp/modules/core/scripts/index/default-importing-controller/parsing-panel.js

```diff
@@ -141,8 +141,9 @@ Refine.DefaultImportingController.prototype._prepareParsingPanel = function() {
 
   var formats = this._job.config.rankedFormats;
   var createFormatTab = function(format) {
+    var formateNames =Refine.importingConfig.formats[format].label;
```

Comment:
There's a typo in `formateNames`, but rather than just fixing that, let's use a variable name which indicates what it contains.
```suggestion
    var formatLabelKey =Refine.importingConfig.formats[format].label;
```
We'll leave `format.label` unchanged for compatibility.